docs/data/material/guides/nextjs/nextjs.md

```diff
@@ -0,0 +1,143 @@
+```
+
+```bash pnpm
+pnpm add @mui/material-nextjs @emotion/cache
+```
+
+</codeblock>
+
+### Configuring
+
+Inside `app/layout.tsx`, import the `AppRouterCacheProvider` and wrap all of the elements under the `<body>`:
+
+```diff
+ // app/layout.tsx
++ import { AppRouterCacheProvider } from '@mui/material-nextjs/v13-appRouter';
+
+ export default function RootLayout(props) {
+   const { children } = props;
+   return (
+     <html lang="en">
+       <body>
++        <AppRouterCacheProvider>{children}</AppRouterCacheProvider>
+       </body>
+     </html>
+   );
+ }
+```
+
+### Using other styling solutions
+
+If you are using a styling solution other than Emotion to customize Material UI components, set `enableCssLayer` to the `options` prop:
+
+```js
+<AppRouterCacheProvider options={{ enableCssLayer: true }}>
+```
+
+The styles generated by Material UI will be wrapped in a CSS `@layer mui` rule, which will be overriden by anonymous layer styles—for example if you are customizing Material UI with CSS modules, Tailwind CSS, or even plain CSS without using `@layer`.
+
+To learn more about the CSS layer, see the [MDN CSS layer documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer).
+
+## Pages Router
+
+The steps below will help you integrate Material UI with the [Pages Router](https://nextjs.org/docs/pages/building-your-application) in Next.js for both [SSR](https://nextjs.org/docs/pages/building-your-application/rendering/server-side-rendering) (server-side rendering) and [SSG](https://nextjs.org/docs/pages/building-your-application/rendering/static-site-generation) (static site generation).
+
+### Intalling the dependencies
+
+You should already have `@mui/material` and `next` installed. Then, install these dependencies:
+
+<codeblock storageKey="package-manager">
+
+```bash npm
+npm install @mui/material-nextjs @emotion/cache @emotion/server
+```
+
+```bash yarn
+yarn add @mui/material-nextjs @emotion/cache @emotion/server
+```
+
+```bash pnpm
+pnpm add @mui/material-nextjs @emotion/cache @emotion/server
+```
+
+</codeblock>
+
+### Configuring
+
+Inside `pages/_document.tsx`:
+
+- import the `documentGetInitialProps` and use it as the Document's getInitialProps.
+- import the `DocumentHeadTags` and render it inside the `<Head>`.
+
+```diff
++ import { DocumentHeadTags, documentGetInitialProps } from '@mui/material-nextjs/v13-pagesRouter';
+
+ export default function MyDocument(props) {
+   return (
+     <Html lang="en">
+       <Head>
++        <DocumentHeadTags {...props} />
+         ...
+       </Head>
+       <body>
+         <Main />
+         <NextScript />
+       </body>
+     </Html>
+   );
+ }
+
++ MyDocument.getInitialProps = documentGetInitialProps;
+```
+
+Then inside `pages/_app.tsx`, import the `AppCacheProvider` and render it as the root element:
+
+```diff
++ import { AppCacheProvider } from '@mui/material-nextjs/pagesRouterV13';
+
+ export default function MyApp(props) {
+   const { Component } = props;
```

Comment:
```suggestion
```

The `Component` destructed from props is not used anywhere, we can omit it for simplicity.